eth/downloader/downloader.go

```diff
@@ -321,12 +321,6 @@ func (d *Downloader) synchronise(id string, hash common.Hash, td *big.Int, mode
 			empty = true
 		}
 	}
-	// Reset any ephemeral sync statistics
-	d.syncStatsLock.Lock()
-	d.syncStatsStateTotal = 0
-	d.syncStatsStateDone = 0
-	d.syncStatsLock.Unlock()
```

Comment:
Just a note: `syncStatsStateTotal` was an unused variable that we can just drop off and `syncStatsStateDone` is better not to be reset so that pulling states in multiple cycles (i.e. reset after some failure) does not look as if it's restarting (causing people to freak out) but rather explicitly display that it's continuing.